apps/chat/src/utils/app/conversation.ts

```diff
@@ -215,9 +211,6 @@ export const isChosenConversationValidForCompare = (
   return true;
 };
 
-export const getOpenAIEntityFullName = (model: DialAIEntity) =>
```

Comment:
why did you remove this method and made a lot of changes instead of just modify it?